kuma/wiki/managers.py

```diff
@@ -41,6 +41,10 @@ def get_by_stale_rendering(self):
 
     def filter_for_list(self, locale=None, tag=None, tag_name=None,
                         errors=None, noparent=None, toplevel=None):
+        """
+        Returns a filtered queryset for a list of names and urls.
+        """
+
         docs = (self.filter(is_redirect=False)
                     .exclude(slug__startswith='User:')
                     .exclude(slug__startswith='Talk:')
```

Comment:
I think `Template_talk` and `Project_talk` can be removed from here.
Did not see `User_talk` anywhere in MDN

Reply:
Here's the page counts for those prefixes:

|Prefix|Count|
|--|--|
|User:|3220|
|Talk:|2531|
|User_talk:|901|
|Template_talk:|17|
|Project_talk:|63|

It can be hard to find them, because we exclude them from the site search.